lib/script.js

```diff
@@ -1,21 +1,33 @@
 /**** To change time from UNIX to normal ***/
 // var time = new Date(timestamp*1000);
 
+var my_location = ""
+var my_city = ""
+var my_state = ""
 var weather_data = [];
 
 $(document).ready(function() {
-    ShowCurrentTime();
-    getWeather("denver", "co");
-    pauseThenLoadWeather();
+    document.querySelector('.dashboard').style.visibility = "hidden";
+    listenForCityChange();
+    
 })
 
 const delay = ms => new Promise(res => setTimeout(res, ms));
 const pauseThenLoadWeather = async () => {
```

Comment:
I really like this function, I can see it adding to user experience.

Reply:
thanks